apps/aetx/test/aetx_tests.erl

```diff
@@ -33,7 +33,8 @@ apply_signed_txs_test_() ->
                               [MinerAccount, AnotherAccount]),
 
                %% Create 3 signed transactions (2 valid, 1 invalid)
-               {ok, CoinbaseTx} = aec_coinbase_tx:new(#{account => MinerPubkey}),
+               {ok, CoinbaseTx} = aec_coinbase_tx:new(#{account => MinerPubkey,
+                                                        block_height => 30}),
```

Comment:
I see this 30 is the same as in https://github.com/aeternity/epoch/pull/796/files#diff-3da268090e9138b3876166f3da939a52R55
May I suggest to use variable in order to ease readability and maintainability?